CRM/Price/BAO/LineItem.php

```diff
@@ -618,7 +618,7 @@ public static function changeFeeSelections(
       $count = count(CRM_Event_BAO_Participant::getParticipantIds($contributionId));
     }
     else {
-      $count = CRM_Utils_Array::value('count', civicrm_api3('MembershipPayment', 'getcount', ['contribution_id' => $contributionId]));
+      $count = civicrm_api3('MembershipPayment', 'getcount', ['contribution_id' => $contributionId]);
```

Comment:
The use of `CRM_Utils_Array::value` was actually masking a bug. `Api3.getcount` returns an int, not an array. So this code was previously broken.
Thanks for flagging that test failure @eileenmcnaughton